### ISSUE TITLE
Use long variable name instead of Rec as parameter name when creating procedures

### ISSUE THREAD

**Issue by wbrakowski** [DavidFeldhoff/al-codeactions#79]
Hi David,

when I create a procedure from a page or table where I have an implicit "Rec", I sometimes want to pass this variable into a new procedure. For example in this PageAction I would like to create the new procedure "CreateOrderFromWebshopOrder".

![image](https://user-images.githubusercontent.com/53928639/89016998-eeaa4200-d319-11ea-9788-386b61b27a8d.png)

The new procedure will have a parameter "Rec: Record "Webshop Sales Header WEB11 CCO".

![image](https://user-images.githubusercontent.com/53928639/89017072-0da8d400-d31a-11ea-9337-641f1d01fde2.png)

Would it be possible to automatically rename this parameter Rec by using the object name? 
Instead of "Rec: Record "Webshop Sales Header WEB11 CCO"----
...this? WebshopSalesHeader: Record "Webshop Sales Header WEB11 CCO"

![image](https://user-images.githubusercontent.com/53928639/89017119-1a2d2c80-d31a-11ea-9c12-0f09f2140b4e.png)

Basically by using the same functionality that is already implemented in the AL Variable Helper when you create a new record:

![image](https://user-images.githubusercontent.com/53928639/89017842-39788980-d31b-11ea-96a5-057ef0ff43d3.png)

But maybe I am the only one who would like to have that :-D

**Comment by DavidFeldhoff** [comment]
Hi Waldemar, thanks for reaching out. I think I have a few other requests which I would add before this one, but I'll have it in mind. I already thought about adding a dependency to that extension (vFull, vShort, and so on). So maybe I'm implementing it with that one as well.. :)

**Comment by DavidFeldhoff** [comment]
Hi Waldemar, I just looked at this issue again and I'm unsure if I should implement it. Maybe it's important if I hand it over to a method (Rec, xRec) if that variable is the current Record or not. If you do not necessarily insist, I would leave it like it is at first, would that be okay?
If more people would like to have that, I'd think about it again.

**Comment by wbrakowski** [comment]
That's fine for me :-)
